source-code-flow-manager-gitlab/src/test/java/com/iprody/source/code/flow/manager/api/job/PipelineJobAPITest.java

```diff
@@ -0,0 +1,68 @@
+package com.iprody.source.code.flow.manager.api.job;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.iprody.source.code.flow.manager.api.job.domain.Job;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.jupiter.api.*;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Flux;
+import reactor.test.StepVerifier;
+
+import java.io.IOException;
+
+class PipelineJobAPITest {
+    private static MockWebServer mockBackEnd;
+    private static ObjectMapper objectMapper;
+    private static PipelineJobAPI jobAPI;
+
+    @BeforeAll
+    static void setUp() throws IOException{
```

Comment:
why do you need before-all instead before-each?
it is better to isolate tests from each other and try to avoid a state sharing, as much as possible

```suggestion
    static void setUp() throws IOException {
```